docs/user-guide/perspective-mode.md

```diff
@@ -1,13 +0,0 @@
-## Perspective Mode
```

Comment:
Shouldn't this be moved somewhere rather than just cut? I think it is a nice capability to talk about.